### ISSUE TITLE
Export functionality for users menu

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#1897]
### Situation Description
Currently there is no export functionality for doing a detailed export of all users on the platform, as part of monitoring and audit. The users menu should give a count of all users on the system
### Feature Description
There should be a detailed export which shows the name of user, user role, state, LGA, health facility, active or inactive, date of creation, and name of admin who created the user, as part of monitoring and evaluation of users created on the system  @MateStrysewskeSym @MartinWahnschaffeSymeda @Chinedar @bernardsilenou @max-hzi @hzi-old

**Comment by Iheanacho2027** [comment]
@foyiri

**Comment by bernardsilenou** [comment]
This issue was raised in the past and we decided not implement such UI because of data security issues. The admin on the server can always export all user accounts from back end.

If needed now, then we have to think of a way of implementing it to be country specific

**Comment by Iheanacho2027** [comment]
@Chinedar  @foyiri

**Comment by bernardsilenou** [comment]
@Iheanacho2027 yes please, its the same thing. I had a meeting today and this point was raided.

**Comment by foyiri** [comment]
Bernard I think for audit pupose we may need to export user accout to
easily check which is active and not and also any further analysis
